### ISSUE TITLE
whisper TensorRT-LLM 是不是还不支持语种识别能力

### ISSUE THREAD

**Issue by Banner-Wang** [k2-fsa/sherpa#687]
我看了相关代码，transcribe时需要指定语言，不然就用默认的。
![image](https://github.com/user-attachments/assets/c700b1be-b328-41a1-8e0f-8e8539447e28)
请问我的理解是否正确？

**Comment by csukuangfj** [comment]
@yuekaizhang  Could you have a look?

**Comment by yuekaizhang** [comment]
@Banner-Wang  You're correct. If you're willing to support the lang id feature with whisper TRT-LLM, I could help you.
